File: api/code-samples/javascript/klaytnDebug/others/setMutexProfileFraction.js
const OpenSdk = require("opensdk-javascript");

(() => {
    const sdk = new OpenSdk(new OpenSdk.ApiClient("https://api.baobab.klaytn.net:8651"));
    
    const number = 2;

    sdk.debug.setMutexProfileFraction(number, {}, (err, data, response) => {
        console.log(data);
    });
}
)()

File: api/code-samples/python/klaytnDebug/others/setMutexProfileFraction.py
import json
from opensdk.sdk import OpenSDK

host = "https://api.baobab.klaytn.net:8651"

number = 2

sdk = OpenSDK(host)
debug_response = sdk.debug.set_mutex_profile_fraction(number)

print(json.loads(debug_response.response.data))
